tasmoadmin/rootfs/etc/nginx/servers/direct.disabled

```diff
@@ -0,0 +1,35 @@
+server {
+    server_name hassio.local;
+    listen 9541 default_server;
+    root /var/www/tasmoadmin/tasmoadmin/;
+    index index.php;
+
+    location /data/firmwares {
+    }
+
+    location ^~ /data/ {
```

Comment:
Note: I added `^~` to this line so that it actually matched. In the current master version I found I was able to access `/data/MyConfig.json` which exposes your credentials.